pkgs/applications/networking/cluster/kops/default.nix

```diff
@@ -65,7 +65,7 @@ rec {
   };
 
   kops_1_20 = mkKops rec {
-    version = "1.20.1";
+    version = "1.20.2";
     sha256 = "sha256-k6ODXbh7Bh+rBw6bjSNLxLY0fz7JLnrmJibnXz5qnSc=";
```

Comment:
the hash also needs to be updated

Reply:
that's the value my machine gave me... I'll try again.

Reply:
For some reason I had to use `nix-prefetch-github` to get (presumably) the correct hash... Both work on my machine somehow.